app/components/listing/ContactInfoTableRows.tsx

```diff
@@ -0,0 +1,59 @@
+import React from "react";
+import { Service } from "../../models";
+
+export const ContactInfoTableRows = ({ service }: { service: Service }) => {
+  const website = service.url || service.resource?.website;
+  const email = service.email || service.resource?.email;
+  const phones = service.resource?.phones;
+
+  if (!website && !phones && !email) {
```

Comment:
Are you checking if `phones` is null here? The `Organization` type suggests `phones` will always be an array. If in experience this value can return null from the api can you update the type please? 

🏓 This array bool check is an alternative I like if you think this list will ever expand: 
```ts
[website, phones, email].every(Boolean)
```